File: functions/quote_generator.js
const quotes = [
  // Add your quotes here
  { quote: "Quote 1", author: "Author 1" },
  { quote: "Quote 2", author: "Author 2" },
  { quote: "Quote 3", author: "Author 3" },
];

exports.handler = async (event, context) => {
  const randomIndex = Math.floor(Math.random() * quotes.length);
  const randomQuote = quotes[randomIndex];

  return {
    statusCode: 200,
    body: JSON.stringify(randomQuote),
  };
};

File: quote_generator.rb
require 'sinatra'
require 'json'

# Load quotes from the JSON file
quotes = JSON.parse(File.read('quotes.json'))

get '/random_quote' do
  content_type :json

  # Select a random quote from the collection
  random_quote = quotes.sample

  # Return the quote as a JSON object
  { quote: random_quote['quote'], author: random_quote['author'] }.to_json
end
